src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -167,7 +168,7 @@ const QDMReporting = () => {
                 Rate Aggregation
               </FieldLabel>
               <FieldSeparator>
-                <FieldInput
+                <TextArea
```

Comment:
1. We have `{...formik.getFieldProps("rateAggregation")}` twice
2. We shouldn't be adding extra styles, I think we can safely remove that.
3. aria-labelledby and type props would also be unnecessary.
4. May be not needed right away but having error and helperText props will help us visualize any formik errors